supplementary_style_guide/master.adoc

```diff
@@ -24,6 +24,9 @@ These recommendations might override existing guidance in _The IBM Style Guide_,
 If applicable, these guidelines provide example formatting in AsciiDoc, which is the markup language that Red Hat Customer Content Services currently uses. However, the guidelines can be applied to any language.
 ====
 
+// Managed cloud services guidelines
+include::style_guidelines/managed-services.adoc[leveloffset=+1]
```

Comment:
@cbudz You have this right under style guidelines (as in, before the first style guideline is shown. Can you put it so that it is done below the all of the style guidelines? As in, after links.adoc is included?